th/2017/winter.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="fr">
+	<head>
+		<title>ฤดูหนาว 2017 | ไทย | The Fansub Database</title>
```

Comment:
Can I add some note for Thai user about the difference between Thai date time & this repository on winter.html?